src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="big-search-form"
+      method="post"
+      name="big"
+      data-qa="big"
+    >
+      <label class="label-big-form">
+        <input
+          class="input-big-form"
+          type="text"
+          name="input-big-form"
+          data-qa="keypress"
+          required placeholder= "Try &quot;Los Angeles&quot;"
```

Comment:
```suggestion
          data-qa="keypress"
          required 
          placeholder="Try &quot;Los Angeles&quot;"
```
required is a separate attribute

Reply:
ready